src/circle.rs

```diff
@@ -24,6 +27,17 @@ impl Circle {
             radius,
         }
     }
+
+    /// Create a circle segment by cutting out parts of this circle.
```

Comment:
Might as well link to `CircleSegment`:

```suggestion
    /// Create a [`CircleSegment`] by cutting out parts of this circle.
    ///
    /// [`CircleSegment`]: struct.CircleSegment.html
```